### ISSUE TITLE
Not supported node in security scheme describedBy

### ISSUE THREAD

**Issue by postatum** [aml-org/amf#490]
### Issue
Running code below should fail or report validation error because according to RAML 1.0 spec provided RAML input is invalid.

The spec says:
> The value of the describedBy node is defined as a map with the following key-value pairs as follows: headers?, queryParameters?, queryString?, responses?, (<annotationName>)?.

Spec link:
https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/#describedby

### AMF Version
4.0.0-SNAPSHOT

### Code
```sh
$ java -jar amf-4.0.0-SNAPSHOT.jar validate -in "RAML 1.0" -mime-in "application/yaml" -p "RAML" file://api.raml
{
  "@type": "http://www.w3.org/ns/shacl#ValidationReport",
  "http://www.w3.org/ns/shacl#conforms": true
}
```

### api.raml
```raml
#%RAML 1.0
title: New API
version: v1
securitySchemes:
  oauth_2_0:
    description: supports OAuth 2.0 for authenticating all API requests.
    type: OAuth 2.0
    describedBy:
      HELLO: 123
```

**Comment by gutee** [comment]
Jira issue to track: https://www.mulesoft.org/jira/browse/APIMF-1522

**Comment by gutee** [comment]
This is fixed and stated for 3.3.0 release
